client/ayon_deadline/repository/readme.md

```diff
@@ -17,14 +17,14 @@ These environment variables are injected into rendering process.
 Deadline triggers the `GlobalJobPreLoad.py` for each Worker as it starts the 
 Job.  
 
-*Note*: It also contains backward compatible logic to preserve functionality 
-for old Pype2 and non-OpenPype triggered jobs.
+<!-- *Note*: It also contains backward compatible logic to preserve functionality 
+for old Pype2 and non-OpenPype triggered jobs. -->
```

Comment:
I commented this part to be able to add it the review in the PR. 
Anyways, should we remove these lines or uncomment them back?

Reply:
yes